libraries/mysql_version.rb

```diff
@@ -27,6 +27,6 @@ def initialize(user, pass)
   end
 
   def mysql_version
-    inspec.command("mysql -sN -e 'SHOW VARIABLES WHERE variable_name = \"version\"'").stdout.strip.split("\t")[1].to_s
+    inspec.command("mysql -sN -e 'SHOW VARIABLES WHERE variable_name = \"version\"'").stdout.strip.split("\t")[1].split("-")[0].to_s
```

Comment:
Agreed. Just replace " with ' around \t and -

Rubocop will be happy.

" means in ruby that the internal string will be interpreted (variables inside will be transformed). ' means in ruby that the internal string will be executed as such (no variable transformation).

Reply:
Thanks, fixed.